api/v3/ActivityType.php

```diff
@@ -110,10 +110,8 @@ function _civicrm_api3_activity_type_create_spec(&$params) {
  *
  * @param array $params
  *   Array including id of activity_type to delete.
- *
- * @return array
- *   API result array
- *
+ * @return array API result array
```

Comment:
Ah - it might be because I have stricter drupal stds in my phpstorm -anyway - it'll do